hazelcast/src/test/java/com/hazelcast/map/nearcache/NearCacheTest.java

```diff
@@ -117,11 +117,18 @@ public void testBasicUsage() {
         }
 
         map.clear();
-        for (HazelcastInstance instance : instances) {
-            NearCache nearCache = getNearCache(mapName, instance);
-            int size = nearCache.size();
-            assertEquals("Near Cache size should be 0 but was " + size, 0, size);
-        }
+
+        assertTrueEventually(new AssertTask() {
```

Comment:
For `assertTrueEventually`: Waiting is needed to see remote node near-caches are also cleared, local near-cache will be cleared already when `map.clear()` returns. means no difference from caller sides' perspective. Also, as of now, member-near-caches' behaviour is inlined with existing client side near-caches.